app/view/show.py

```diff
@@ -11,8 +11,33 @@
 from app.pacman import get_pkg
 from app.util import chunks, multiline_to_list
 from collections import defaultdict
+from urllib.parse import urlencode
 
 
+def link_encode(url, query):
```

Comment:
This function should be obsolete when calling urlencode() from within the jinja2 templateing